omnigibson/robots/franka.py

```diff
@@ -139,9 +228,18 @@ def _default_controllers(self):
         controllers["gripper_{}".format(self.default_arm)] = "MultiFingerGripperController"
         return controllers
 
+    @property
+    def _default_gripper_multi_finger_controller_configs(self):
+        conf = super()._default_gripper_multi_finger_controller_configs
+        # If the end effector is not a gripper, set the mode to independent
+        if self.end_effector != "gripper":
+            conf[self.default_arm]["mode"] = "independent"
+            conf[self.default_arm]["command_input_limits"] = None
+        return conf
+
     @property
     def _default_joint_pos(self):
-        return np.array([0.00, -1.3, 0.00, -2.87, 0.00, 2.00, 0.75, 0.00, 0.00])
+        return self.__default_joint_pos
```

Comment:
nit, maybe we can rename this to something a bit more descriptive, e.g.: `self._default_robot_model_joint_pos` or something?